networks/cardano/tokens/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b.json

```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "snek",
+    "decimals": 6,
```

Comment:
Snek has decimals 0: https://cardanoscan.io/token/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b

Reply:
Confirm this with a few sources:
1. https://cardanoassets.com/asset108xu02ckwrfc8qs9d97mgyh4kn8gdu9w8f5sxk
2. https://minswap.org/tokens/snek
3. https://adastat.net/tokens/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b